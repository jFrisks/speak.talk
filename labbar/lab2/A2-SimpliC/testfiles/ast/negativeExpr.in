int g() {
  return ---3;
}