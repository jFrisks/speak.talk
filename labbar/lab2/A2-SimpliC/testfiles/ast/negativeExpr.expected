Program
  List
    FunctionDecl ID="g"
      List
      Block
        List
          ReturnStmt
            Neg
              Numeral NUMERAL="3"